documentation/byo-secrets.md

```diff
@@ -49,114 +49,61 @@ only run if they have access to the secrets they will need.
 
 # How it works
 
-Tests are executed if `*_DOMAIN` exists.  If the value is empty or
-unset, the test is skipped.  If a test doesn't require secrets, the
-`*_DOMAIN` variable is hardcoded so that it always runs.  Otherwise, it is set by looking up
-the secret. For example, if a provider is called `FANCYDNS`, there must
+Tests are executed if `*_DOMAIN` exists where `*` is the name of the provider.  If the value is empty or
+unset, the test is skipped.  
+For example, if a provider is called `FANCYDNS`, there must
```

Comment:
FYI:
GHA has "secrets" and "variables".  The `*_DOMAIN` item must b a variable.  All the others must be secrets.